rust/src/domains/polars/expr/mod.rs

```diff
@@ -0,0 +1,214 @@
+use polars::lazy::dsl::Expr;
+use polars::prelude::*;
+use std::collections::BTreeSet;
+use std::fmt::{Debug, Formatter};
+
+use crate::core::{Metric, MetricSpace};
+use crate::metrics::{
+    AbsoluteDistance, ChangeOneDistance, HammingDistance, InsertDeleteDistance, LInfDistance,
+    LpDistance, PartitionDistance, SymmetricDistance,
+};
+use crate::traits::ProductOrd;
+use crate::transformations::DatasetMetric;
+use crate::{core::Domain, error::Fallible};
+
+use super::{Frame, FrameDomain, LogicalPlanDomain, NumericDataType, SeriesDomain};
+
+#[cfg(feature = "ffi")]
+mod ffi;
+
+// TODO: remove this allow marker later
```

Comment:
Link to issue so we don't lose track of this?

Reply:
I'll just leave it as a warning for now.